doc/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Do you want to manually edit it back to what it was? Cause we probably don't want to flip this number all the time. It's hacky so I don't insist. Just an idea...

Reply:
Done.